.github/workflows/pythonapp.yml

```diff
@@ -24,8 +24,8 @@ jobs:
     - run: conda --version
     - run: python --version
     - run: |
-        conda install -c conda-forge h5py rdkit
- 
+        conda install -c conda-forge h5py rdkit nbsphinx pandoc ipython
```

Comment:
Installing `nbsphinx` was suprinsgly tricky.
It seems that if you're using conda then you **will** have to install `nbsphinx` and the likes via conda as well.

Reply:
Wouldn't be better to directly install `jupyter` install of ipython?